ansible/roles/docker/destroy-container/tasks/main.yml

```diff
@@ -1,11 +0,0 @@
----
-- name: Stopping the container
```

Comment:
why have you removed tasks from destroy-container role?

Reply:
I have removed the entire destroy-container role. It was unnecessary because ansible docker-container can automatically destroy and re-create docker containers with option recreate: yes. There was no benefit in having a destroy-container role.

Reply:
https://docs.ansible.com/ansible/latest/modules/docker_container_module.html